integration_tests/integration/barcode/createBarcodeAddressImage.spec.ts

```diff
@@ -24,14 +25,27 @@ context('Create Barcode Image', () => {
     page.continueToImageErrors().hasErrorContaining('error')
   })
 
-  // TODO - the intent of the test is correct, but wiremock stubs will need creating once the barcode API is called with recipient details (SLM-99)
-  // At the moment we cannot reliably tell wiremock to fail for the 2nd call
-  it.skip('should show an error if creating the first barcode succeeds but subsequent barcodes fail', () => {
```

Comment:
Also took the opportunity to address this todo seeing as I was working in this test class anyway, and we've completed SLM-99 👍